js/apps/system/_admin/aardvark/APP/react/src/views/analyzers/addAnalyzer/AddAnalyzerModal.tsx

```diff
@@ -21,7 +21,7 @@ const INITIAL_VALUES = {
   type: "identity",
   features: [],
   properties: {}
-} as AnalyzerDescription;
+} as CreateAnalyzerOptions & { name: string };
```

Comment:
I'm thinking this `name` should be a part of the `CreateAnalyzerOptions` in `arangojs` itself. What do you think?

Reply:
nevermind, I think it's it's obvious from the usage. Resolving this for now :)